sub-packages/bionemo-scdl/src/bionemo/scdl/io/single_cell_memmap_dataset.py

```diff
@@ -480,10 +512,6 @@ def load_h5ad(
 
         self.dtypes[f"{FileNames.DATA.value}"] = count_data.dtype
```

Comment:
This is mutated since the data could theoretically be in a different format. We only know what that is once we in the anndata.

Reply:
It can just be passed as a parameter. There isn't a need to have it be an attribute of the class. Something to think about for refactoring at a later date.